src/scripts/Sitemap.js

```diff
@@ -4,7 +4,15 @@ import SelectorList from './SelectorList';
 import Model from './Model';
 
 export default class Sitemap {
-	constructor(id, startUrls, model, selectors) {
+	constructor(id, startUrls, model, selectors, extensionVersion = null) {
+		this.rootSelector = { id: '_root', uuid: '0' };
+		this.extensionVersion =
+			extensionVersion !== null
+				? extensionVersion
+				: (selectors.length > 0 && selectors[0].uuid !== undefined) ||
+				  selectors.length === 0
+				? 1
+				: 0;
```

Comment:
Do not guess version by sitemap content. In future there will be more versions, guessing them would complicate the code.